tooling/maven/archetypes/multimodule/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>${groupId}</groupId>
+    <artifactId>${rootArtifactId}</artifactId>
+    <version>${version}</version>
+    <packaging>pom</packaging>
+
+    <properties>
+        <!-- plugin versions to use in generated archetypes -->
+        <scrplugin.version>1.21.0</scrplugin.version>
+        <bundleplugin.version>3.0.0</bundleplugin.version>
+        <compilerplugin.version>3.3</compilerplugin.version>
+        <slingplugin.version>2.1.2</slingplugin.version>
+        <launchpadplugin.version>2.3.2</launchpadplugin.version>
+        <jarplugin.version>2.6</jarplugin.version>
+        <warplugin.version>2.6</warplugin.version>
+        <jettyplugin.version>6.1.23</jettyplugin.version>
+        
+        <!-- various properties which influence generated archetypes -->
+        <archetype.java.version>7</archetype.java.version>
+        
+        <!-- dependency versions to use in generated archetypes -->
+        <junit.version>4.12</junit.version>
+        <scrannotations.version>1.9.12</scrannotations.version>
+        <slingapi.version>2.9.0</slingapi.version>
```

Comment:
this is a really old api version, we are already at 2.16.2:
https://github.com/apache/sling/blob/trunk/bundles/api/pom.xml